dtm0/ut/main.c

```diff
@@ -109,7 +116,41 @@ static void dtm0_ut_send_fops(struct m0_rpc_session *cl_rpc_session)
 	M0_ASSERT(m0_dtm0_ts__invariant(&reply_data.dti_ts));
 
 	M0_UT_ASSERT(m0_dtm0_tid_cmp(&dcs, &txr.dtd_id, &reply_data) == M0_DTS_EQ);
+	m0_fop_put_lock(fop);
```

Comment:
Let's see if we can move this code (send P msg, recv it, send P msg back) into its own test case. This is optional, we can address it later.

Reply:
Yes, I will make a note in the code and address it later.